cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -17,15 +17,19 @@ extensions:
       es_main:
         server_urls: http://localhost:9200
         log_level: "error"
-        index_prefix: "jaeger-main"
-        use_aliases: false
-        create_mappings: true
+        index:
+          prefix: "jaeger-main"
+        aliases:
+          enabled: false
+          create_mappings: true
```

Comment:
this looks confusing to me - when something is `enabled: false` I don't expect any other properties to make sense. Is `create_mappings` even related to aliases?